src/control/server/mgmt_pool.go

```diff
@@ -619,6 +619,19 @@ func resolvePoolPropVal(req *mgmtpb.PoolSetPropReq) (*mgmtpb.PoolSetPropReq, err
 		default:
 			return nil, errors.Errorf("unhandled self_heal type %q", healType)
 		}
+	// case "policy":
+	// 	newReq.SetPropertyNumber(drpc.PoolPropertyPolicy)
+
+	// 	recType := strings.TrimSpace(req.GetStrval())
+	// 	switch strings.ToLower(recType) {
+	// 	case "io_size":
+	// 		newReq.SetValueNumber(uint64(drpc.PoolPolicyIoSize))
+	// 	case "write_intensivity":
+	// 		newReq.SetValueNumber(uint64(drpc.PoolPolicyWriteIntensivity))
+	// 	default:
+	// 		return nil, errors.Errorf("unhandled policy type %q", recType)
+	// 	}
+
```

Comment:
???

Reply:
This was commented out due to being unsupported when used with struct (i.e. dpe_val_ptr) based properties